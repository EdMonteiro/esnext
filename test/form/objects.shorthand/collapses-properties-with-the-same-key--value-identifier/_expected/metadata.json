{
  "objects.shorthand": {
    "properties": [
      {
        "type": "ObjectProperty",
        "computed": false,
        "key": {
          "type": "Identifier",
          "name": "a",
          "typeAnnotation": null
        },
        "value": {
          "type": "Identifier",
          "name": "a",
          "typeAnnotation": null
        },
        "shorthand": false,
        "decorators": null
      }
    ]
  }
}