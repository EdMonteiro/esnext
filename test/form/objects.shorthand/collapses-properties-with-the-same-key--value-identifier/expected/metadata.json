{
  "objects.shorthand": {
    "properties": [
      {
        "type": "Property",
        "kind": "init",
        "computed": false,
        "shorthand": false,
        "method": false,
        "key": {
          "type": "Identifier",
          "name": "a"
        },
        "value": {
          "type": "Identifier",
          "name": "a"
        }
      }
    ]
  }
}